Commands/Commands/utils.js

```diff
@@ -570,6 +592,12 @@ function wait(bot, msg) {
   })
 }
 
+function delay(delayMS) {
+  return new Promise((resolve) => {
+    setTimeout(() => resolve(arg), delayMS);
```

Comment:
Whoops, that's because I removed the function return. I'm trying to write this code blind, as I haven't got my own instance up yet